docs/modules/ROOT/pages/large/multi-file-builds.adoc

```diff
@@ -13,4 +13,7 @@ include::partial$example/large/multi/11-helper-files.adoc[]
 
 include::partial$example/large/multi/12-helper-files-sc.adoc[]
 
+== Multi Language Project
+
+include::partial$example/large/multi/14-multi-language.adoc[]
```

Comment:
Let's move this to a separate `large/multi-language-builds.adoc` page, and include it in the `nav.adoc`